.github/FUNDING.yml

```diff
@@ -0,0 +1,3 @@
+# These are supported funding model platforms
```

Comment:
You are removing this file?

Reply:
thanks for catching that and I removed it